fenix/app/src/main/java/org/mozilla/fenix/addons/ui/AddonPermissionsModifyWebsiteDataSection.kt

```diff
@@ -0,0 +1,114 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.addons.ui
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.lazy.LazyColumn
+import androidx.compose.foundation.lazy.items
+import androidx.compose.material.Divider
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.semantics.heading
+import androidx.compose.ui.semantics.semantics
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import org.mozilla.fenix.R
+import org.mozilla.fenix.compose.SwitchWithLabel
+import org.mozilla.fenix.compose.list.TextListItem
+import org.mozilla.fenix.theme.FirefoxTheme
+
+/**
+ * Displays all the websites that have the permissions granted
+ *
+ * @websites the list of websites that have granted permission
+ */
+@Composable
+fun SectionModifyWebsiteData(websites: List<String>) {
```

Comment:
Nit: Maybe `OriginsSection` to match the APIs https://github.com/mozilla-mobile/firefox-android/pull/5793/files#diff-26e01436aa717b89aba5b097f97aca6777693d162bcf030f28006165c6998414R64

Reply:
nit: Instead of `websites` we could rename to `origins`, in the final API this will be a `List<Permission>` see https://github.com/mozilla-mobile/firefox-android/pull/5793/files#diff-26e01436aa717b89aba5b097f97aca6777693d162bcf030f28006165c6998414R64-R65